packages/phone-number-privacy/src/config.ts

```diff
@@ -20,6 +19,13 @@ interface Config {
     database: string
     host: string
   }
+  keyVault: {
+    azureClientID: string
+    azureClientSecret: string
+    azureTenant: string
```

Comment:
i only see `azureVaultName` and `azureSecretName` used, where do these fields come into play?

Reply:
Good catch! I forgot to set these as env variables for the `AzureKeyVaultClient` to use them.